### DISCUSSION TITLE
Importing and Attaching Multimedia to the Catalogue

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#331] [HOW TO...]
**Used By**: All
**In Use?**: Yes

[Googledoc workflow](https://docs.google.com/document/d/1T42fcRV3r8paQHZw_00M2cwn2dmzaJpEJPZzorxtmec)

**Comment by magpiedin** [comment]
(matches FAQ https://github.com/fieldmuseum/EMu-Documentation/discussions/364)
